spacy/kb.pxd

```diff
@@ -1,36 +1,20 @@
 """Knowledge-base for entity or concept linking."""
-from cymem.cymem cimport Pool
 from preshed.maps cimport PreshMap
 from libcpp.vector cimport vector
 from libc.stdint cimport int32_t, int64_t
 from libc.stdio cimport FILE
 
-from .vocab cimport Vocab
 from .typedefs cimport hash_t
 from .structs cimport KBEntryC, AliasC
-
+from .kb_base cimport BaseKnowledgeBase
```

Comment:
`BaseKnowledgeBase` and `KnowledgeBase` are placeholders - we can find better names, I think.

Reply:
Internal proposal: `InMemoryLookupKB`